src/System.Private.Interop/src/System.Private.Interop.csproj

```diff
@@ -44,45 +44,73 @@
     <Compile Include="Shared\ComException.cs" />
     <Compile Include="Shared\Marshal.cs" />
     <Compile Condition="'$(EnableWinRT)'=='true'" Include="Shared\WindowsRuntimeMarshal.cs" />
-    <Compile Include="System\Runtime\InteropServices\AllowReversePInvokeCallsAttribute.cs" />
+    <Compile Include="..\..\System.Private.CoreLib\shared\System\Runtime\InteropServices\AllowReversePInvokeCallsAttribute.cs">
```

Comment:
These moved files should be in one block under `Condition="'$(IsProjectNLibrary)' == 'true'"`

Reply:
Actually, here is a better plan:

- Delete the files from this .csproj
- Include them in CoreLib .csproj unconditionally
- Add the moved types to ttps://github.com/dotnet/corert/blob/master/src/System.Private.Interop/src/TypeForwarders.cs